src/lib/OpenEXR/ImfStandardAttributes.cpp

```diff
@@ -68,34 +68,60 @@ using namespace std;
 IMF_STD_ATTRIBUTE_IMP (adoptedNeutral, AdoptedNeutral, V2f)
-IMF_STD_ATTRIBUTE_IMP (renderingTransform, RenderingTransform, string)
-IMF_STD_ATTRIBUTE_IMP (lookModTransform, LookModTransform, string)
-IMF_STD_ATTRIBUTE_IMP (xDensity, XDensity, float)
 IMF_STD_ATTRIBUTE_IMP (owner, Owner, string)
+IMF_STD_ATTRIBUTE_IMP (creator, Creator, std::string)
 IMF_STD_ATTRIBUTE_IMP (comments, Comments, string)
 IMF_STD_ATTRIBUTE_IMP (capDate, CapDate, string)
 IMF_STD_ATTRIBUTE_IMP (utcOffset, UtcOffset, float)
 IMF_STD_ATTRIBUTE_IMP (longitude, Longitude, float)
 IMF_STD_ATTRIBUTE_IMP (latitude, Latitude, float)
 IMF_STD_ATTRIBUTE_IMP (altitude, Altitude, float)
+IMF_STD_ATTRIBUTE_IMP (cameraIdentifier, CameraIdentifier, std::string)
+IMF_STD_ATTRIBUTE_IMP (cameraLabel, CameraLabel, std::string)
+IMF_STD_ATTRIBUTE_IMP (cameraMake, CameraMake, std::string)
+IMF_STD_ATTRIBUTE_IMP (cameraModel, CameraModel, std::string)
+IMF_STD_ATTRIBUTE_IMP (cameraSerialNumber, CameraSerialNumber, std::string)
+IMF_STD_ATTRIBUTE_IMP (cameraFirmwareVersion, CameraFirmwareVersion, std::string)
+IMF_STD_ATTRIBUTE_IMP (isoSpeed, IsoSpeed, float)
+IMF_STD_ATTRIBUTE_IMP (lensMake, LensMake, std::string)
+IMF_STD_ATTRIBUTE_IMP (lensModel, LensModel, std::string)
+IMF_STD_ATTRIBUTE_IMP (lensSerialNumber, LensSerialNumber, std::string)
+IMF_STD_ATTRIBUTE_IMP (lensAttributes, LensAttributes, std::string)
 IMF_STD_ATTRIBUTE_IMP (focus, Focus, float)
-IMF_STD_ATTRIBUTE_IMP (expTime, ExpTime, float)
+IMF_STD_ATTRIBUTE_IMP (focalLength, FocalLength, float)
 IMF_STD_ATTRIBUTE_IMP (aperture, Aperture, float)
-IMF_STD_ATTRIBUTE_IMP (isoSpeed, IsoSpeed, float)
-IMF_STD_ATTRIBUTE_IMP (envmap, Envmap, Envmap)
+IMF_STD_ATTRIBUTE_IMP (convergenceDistance, ConvergenceDistance, float)
+IMF_STD_ATTRIBUTE_IMP (interocularDistance, InterocularDistance, float)
+IMF_STD_ATTRIBUTE_IMP (multiView, MultiView, StringVector)
+IMF_STD_ATTRIBUTE_IMP (recorderMake, RecorderMake, std::string)
+IMF_STD_ATTRIBUTE_IMP (recorderModel, RecorderModel, std::string)
+IMF_STD_ATTRIBUTE_IMP (recorderSerialNumber, RecorderSerialNumber, std::string)
+IMF_STD_ATTRIBUTE_IMP (recorderFirmwareVersion, RecorderFirmwareVersion, std::string)
+IMF_STD_ATTRIBUTE_IMP (storageMediaSerialNumber, StorageMediaSerialNumber, std::string)
+IMF_STD_ATTRIBUTE_IMP (reelName, ReelName, std::string)
+IMF_STD_ATTRIBUTE_IMP (uuid, Uuid, std::string)
 IMF_STD_ATTRIBUTE_IMP (keyCode, KeyCode, KeyCode)
+IMF_STD_ATTRIBUTE_IMP (expTime, ExpTime, float)
+IMF_STD_ATTRIBUTE_IMP (captureRate, CaptureRate, Rational)
+IMF_STD_ATTRIBUTE_IMP (timecodeRate, TimecodeRate, int)
 IMF_STD_ATTRIBUTE_IMP (timeCode, TimeCode, TimeCode)
-IMF_STD_ATTRIBUTE_IMP (wrapmodes, Wrapmodes, string)
+IMF_STD_ATTRIBUTE_IMP (imageCounter, ImageCounter, int)
 IMF_STD_ATTRIBUTE_IMP (framesPerSecond, FramesPerSecond, Rational)
-IMF_STD_ATTRIBUTE_IMP (multiView, MultiView, StringVector)
+IMF_STD_ATTRIBUTE_IMP (cameraPosition, CameraPosition, IMATH_NAMESPACE::V3f)
```

Comment:
We're in the right namespace already, I think you can just say V3f, just  a few lines below it says M44f.

Reply:
unlike `string` vs `std::string` I would argue that there's no ambiguity about V3f, so V3f is fine as you suggest.

Reply:
Cary removed the namespacing